### DISCUSSION TITLE
Change opening icon

### DISCUSSION THREAD

**Discussion by GabrieleTorricella** [dotnet/maui#3567] [Q&A] [answered]
I want to change the opening icon.
I want to specify my request... At the moment, when I start the app on a device, there's wrote ".Net" for something like 5sec (like when you open Instagram there's wrote "Instagram" and something else). How can I change that? I didn't found anything and I downloaded also the examples from here but I didn't find how change it

Thank you!

**Comment by nogginbox** [comment, marked as answer]
I've not tried changing this, but you should be able to change the splash screen (which is the opening screen) in the .csproj file.

It's all specified in this ItemGroup and you should change ```MauiSplashScreen```:
```
<ItemGroup>
    <!-- App Icon -->
    <MauiIcon Include="Resources\appicon.svg" ForegroundFile="Resources\appiconfg.svg" Color="#512BD4" />

    <!-- Splash Screen -->
    <MauiSplashScreen Include="Resources\appiconfg.svg" Color="#512BD4" />

    <!-- Images -->
    <MauiImage Include="Resources\Images\*" />

    <!-- Custom Fonts -->
    <MauiFont Include="Resources\Fonts\*" />
</ItemGroup>
```

**Reply by GabrieleTorricella** [reply]
It works! Thank you

**Reply by Alfredsmarteze** [reply]
Hello, is it that is not possible for now to change the maui app icon? I have been trying to change it but is not changing
